File: python/08_is_multiple.py
"""¿Es múltiplo? — Fácil · Números

Devuelve True si n es múltiplo de m. Si m es 0, devuelve False.

Ejemplos: is_multiple(10, 2) -> True, is_multiple(7, 2) -> False.

Fuente: https://coding-challenges.dev/problems/python-es-multiplo
"""


def is_multiple(n: int, m: int) -> bool:
    return m != 0 and n % m == 0


print(is_multiple(10, 2))
print(is_multiple(7, 2))

File: typescript/08-is-multiple.ts
// ¿Es múltiplo? — Fácil · Números
// Devuelve true si n es múltiplo de m. Si m es 0, devuelve false.
// Fuente: https://coding-challenges.dev/problems/es-multiplo

function isMultiple(n: number, m: number): boolean {
  return m !== 0 && n % m === 0;
}

// No modificar: necesario para evaluar el resultado.
export { isMultiple };
